misc/kind/cluster.yaml

```diff
@@ -187,3 +192,20 @@ nodes:
       materialize.cloud/disk: true
       materialize.cloud/availability-zone: "3"
       topology.kubernetes.io/zone: "3"
+  - role: worker
+    labels:
+      materialize.cloud/disk: true
+      environmentd: true
+      materialize.cloud/availability-zone: "3"
+      topology.kubernetes.io/zone: "3"
+  - role: worker
+    labels:
+      materialize.cloud/disk: true
+      environmentd: true
+      materialize.cloud/availability-zone: "3"
+      topology.kubernetes.io/zone: "3"
+  - role: worker
+    labels:
+      supporting-services: true
```

Comment:
Other services that need to be scheduled on the supporting services :
- debezium
- minio

as I guess a MySQL service may also come at some point, it may be best to require a `is_supporting_service` argument / method to `K8sService` so that each new service is forced to declare where it stands. Or assume that everything is a supporting service except those that are not?